src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -90,17 +90,18 @@ public UIElement(string htmlTag = "div", bool isSvg = false)
 				htmlId: HtmlId,
 				htmlTag: HtmlTag,
 				handle: Handle,
-				fullName: type.FullName,
+				uiElementRegistrationId: UIElementNativeRegistrar.GetForType(type),
 				htmlTagIsSvg: HtmlTagIsSvg,
-				isFrameworkElement: _isFrameworkElement,
-				isFocusable: false,
-				classes: ClassNames.GetForType(type)
+				isFocusable: false
 			);
 
 			InitializePointers();
 			UpdateHitTest();
 
-			FocusManager.Track(this);
+			if (this is FrameworkElement)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Offload the code that's conditional on this 'is' test to the appropriate subclass and remove the test.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)